ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -3672,7 +3672,17 @@ private static Protection GetProtection(StyleInfo styleInfo)
             };
         }
 
-        private static bool CellFormatsAreEqual(CellFormat f, StyleInfo styleInfo)
+        /// <summary>
+        /// Check if two style are equivalent.
+        /// </summary>
+        /// <param name="f">Style in the OpenXML format.</param>
+        /// <param name="styleInfo">Style in the ClosedXML format.</param>
+        /// <param name="compareAlignment">Flag specifying whether or not compare the alignments of two styles.
```

Comment:
Any other properties that act in this way, besides alignment?

Reply:
From what I know, other properties acts the same way in `x:cellStyleXfs` and `x:cellXfs`.